src/continuous-scroll.js

```diff
@@ -0,0 +1,202 @@
+/* global TweenLite, TimelineLite, Linear */
+import React from "react";
+import "gsap";
+
+let tween = null,
+  timeline = null;
+
+const ContinuousScroll = React.createClass({
+  propTypes: {
+    type: React.PropTypes.oneOf(["none", "continuous", "page"]),
+    speed: React.PropTypes.oneOf(["slow", "medium", "fast"]),
+    pause: React.PropTypes.number,
+    height: React.PropTypes.number.isRequired,
+    data: React.PropTypes.array.isRequired,
+    isRefreshing: React.PropTypes.bool,
```

Comment:
@donnapep Can you clarify why `isRefreshing` is a prop on the component and not a state that it internally manages? Do you have an example of when it would be used? Thanks